range_buckets.test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import range_buckets
+
+
+class RangeBucketTest(unittest.TestCase):
+    def test_range_details(self):
+        self.assertTrue(range_buckets.getRangeDetails([4, 5]) == 1)
+        self.assertTrue(range_buckets.getRangeDetails([2, 3, 4, 6, 7]) == 2)
+        self.assertTrue(range_buckets.getRangeDetails([1, 7, 2, 3, 4, 6]) == 2)
+
+    def test_print_range_details(self):
+        self.assertTrue(range_buckets.printRangeDetails([4, 5]) == 1)
+        self.assertTrue(range_buckets.printRangeDetails(
```

Comment:
These tests do not check the `firstNumber` and `endNumber`. Consequence is that manual test needs to be done, and a new developer would find it hard to recognize mistakes after changing the code.
Hint: Separate the range computation from printing.